data/themes/darktable.css

```diff
@@ -979,10 +979,16 @@ notebook tabs,
 #blending-tabs #dt-button
 {
 	font-size: 0.8em;
-	padding: 0.1em;
+	padding: 0.25em;
 	margin: 0 0.5em;
 }
 
+#blending-tabs #dt-button,
+#modules-tabs #dt-button
+{
+	min-width: 1.25em;
```

Comment:
It is still visually smaller, I tried 1.4 and seems better to me.

Reply:
I disagree. This button needs to be a little smaller than main buttons.